src/App.js

```diff
@@ -0,0 +1,56 @@
+// import {
+//   BrowserRouter as Router,
+//   Routes,
+//   Route,
+// } from 'react-router-dom';
+// import { Provider, useSelector } from 'react-redux';
+// import Bookss from './Components/Bookss';
+// import store from './Redux/configureStore';
+// import Navbar from './Components/Navbar';
+// import Catergories from './Components/Catergories';
+
+// function App() {
+//   const bookss = useSelector((state) => state.BookReducer);
+//   return (
+//     <Router className="route">
+//       <Navbar />
+//       <Provider store={store}>
+//         <Routes>
+//           <Route path="/" exact element={<Bookss books={bookss} />} />
+//           <Route path="/catergories" element={<Catergories />} />
+//         </Routes>
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments. 👍🏻